src/types/ajv/CycleRecordSchema.ts

```diff
@@ -0,0 +1,346 @@
+import { addSchema, addSchemaDependency } from '../../utils/serialization/SchemaHelpers'
+import { AJVSchemaEnum } from '../enum/AJVSchemaEnum'
+
+// AJV Schema for CycleRecord
+let schemaCycleRecord: object | undefined = {
+  type: 'object',
+  properties: {
+    networkId: { type: 'string' },
+    counter: { type: 'number' },
+    previous: { type: 'string' },
+    start: { type: 'number' },
+    duration: { type: 'number' },
+    networkConfigHash: { type: 'string' },
+    mode: {
+      type: 'string',
+      enum: ['forming', 'processing', 'safety', 'recovery', 'restart', 'restore', 'shutdown'],
+    },
+    safetyMode: { type: 'boolean' },
+    safetyNum: { type: 'number' },
+    networkStateHash: { type: 'string' },
+    refreshedArchivers: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          publicKey: { type: 'string' },
+          ip: { type: 'string' },
+          port: { type: 'number' },
+          curvePk: { type: 'string' },
+        },
+        required: ['publicKey', 'ip', 'port', 'curvePk'],
+      },
+    },
+    refreshedConsensors: {
```

Comment:
this is wrong.. refreshedConsensors would correct when : 
![image](https://github.com/user-attachments/assets/a4712118-d5be-43b5-b640-5ea498d83750)